### ISSUE TITLE
Install pytorch (1.11.0+cu113) and pytorch-geometric (2.0.4) environment in HPC 

### ISSUE THREAD

**Issue by QingbiaoLi** [pyg-team/pytorch_geometric#5153]
### 😵 Describe the installation problem

I started using pytorch-geometric (https://pytorch-geometric.readthedocs.io/) in HPC.  I am using pytorch (1.11.0+cu113) and pytorch-geometric (2.0.4).
However, I found either is one issue in using it on HPC (https://www.hpc.cam.ac.uk/). I can start the training model in CPU model, but the progress never starts as shown in attachment 1.
If I start to train the model in GPU but it results in one issue in the library of shown in the GitHub request or attachment 2.
https://github.com/pyg-team/pytorch_geometric/issues/2429
I have tried to build very thing from source, but it still doesn't work.
https://pytorch-geometric.readthedocs.io/en/latest/notes/installation.html#installation-from-source
Do you have any clue in how I can solve it?

![Issue_when_run_GPU](https://user-images.githubusercontent.com/10966548/183265413-30c1ee1a-c1e6-4629-9b21-74ba46ab1ac1.png)
![Issue_when_run_CPU](https://user-images.githubusercontent.com/10966548/183265414-1c692090-759a-42c7-93e5-febbc98a4872.png)

### Environment

* PyG version: 2.0.4
* PyTorch version: 1.11.0+cu113
* OS: 
LSB Version:	:core-4.1-amd64:core-4.1-noarch:cxx-4.1-amd64:cxx-4.1-noarch:desktop-4.1-amd64:desktop-4.1-noarch:languages-4.1-amd64:languages-4.1-noarch:printing-4.1-amd64:printing-4.1-noarch
Distributor ID:	Scientific
Description:	Scientific Linux release 7.9 (Nitrogen)
Release:	7.9
Codename:	Nitrogen
* Python version: 3.8

* CUDA/cuDNN version: cuda11.3-cudnn8
* How you installed PyTorch and PyG (`conda`, `pip`, source): I have tried conda, and pip
* Any other relevant information (*e.g.*, version of `torch-scatter`):

**Comment by QingbiaoLi** [comment]
I tried to install it from Conda after I got GPU node on HPC. But it caused another issue.

![image](https://user-images.githubusercontent.com/10966548/183778858-a4c12a2c-fbda-4593-828c-f926947558c7.png)

**Comment by rusty1s** [comment]
I see. This will be fixed in an upcoming PyG release where `torch-spline-conv` is now an optional dependency when installing from conda. Eventually, just removing the `torch-spline-conv` dependency resolves this.

**Comment by QingbiaoLi** [comment]
Nice! May I ask when it will roughly be released?

**Comment by rusty1s** [comment]
I am targeting next Wednesday.

**Comment by QingbiaoLi** [comment]
Amazing. Thank you so much! Look forward to it!

**Comment by QingbiaoLi** [comment]
Hello,
May I ask whether it has released? If so, which version of Pytorch Geo?

**Comment by rusty1s** [comment]
We released PyG 2.1.0 a few hours ago. Still need to update the `conda` package though!

**Comment by rusty1s** [comment]
Hi @QingbiaoLi Can you do me a favor and manually fix PyG on your end by replacing [this](https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/nn/conv/spline_conv.py#L17) line with
```
except ImportError, OSError:
```

**Comment by rusty1s** [comment]
Added this to master as well: https://github.com/pyg-team/pytorch_geometric/pull/5276

**Comment by QingbiaoLi** [comment]
> Hi @QingbiaoLi Can you do me a favor and manually fix PyG on your end by replacing [this](https://github.com/pyg-team/pytorch_geometric/blob/master/torch_geometric/nn/conv/spline_conv.py#L17) line with
> 
> ```
> except (ImportError, OSError):
> ```

Hello @rusty1s Thanks for the reply. May I ask how I should change it? Your script looks the same as the one in your link.

**Comment by rusty1s** [comment]
Yes, because I pushed that change to master. You can patch it locally within the `site-packages` directory.

**Comment by QingbiaoLi** [comment]
> Yes, because I pushed that change to master. You can patch it locally within the `site-packages` directory.

Thanks! It works now!

However, I got the following warning. I wonder whether it will be a problem.

![image](https://user-images.githubusercontent.com/10966548/186638358-3fa02473-6463-4a0f-b51a-e35b0d38260c.png)

**Comment by rusty1s** [comment]
No, this is just a deprecation warning present in PyG 2.0 in case you are operating on PyTorch 1.12 - it will work as expected.
